ffs/api/api.go

```diff
@@ -48,18 +48,28 @@ type API struct {
 
 // New returns a new Api instance.
 func New(ctx context.Context, iid ffs.APIID, is InstanceStore, sch ffs.Scheduler, wm ffs.WalletManager, dc ffs.DefaultCidConfig) (*API, error) {
+	addr, err := wm.NewAddress(ctx, defaultAddressType)
+	if err != nil {
+		return nil, fmt.Errorf("creating new wallet addr: %s", err)
+	}
+
+	dc.Cold.Filecoin.Addr = addr
+
 	if err := dc.Validate(); err != nil {
 		return nil, fmt.Errorf("default cid config is invalid: %s", err)
 	}
-	addr, err := wm.NewAddress(ctx, defaultWalletType)
-	if err != nil {
-		return nil, fmt.Errorf("creating new wallet addr: %s", err)
+
+	addrInfo := AddrInfo{
+		Name: "Initial Address",
+		Addr: addr,
 	}
+
 	config := Config{
 		ID:               iid,
-		WalletAddr:       addr,
+		Addrs:            map[string]AddrInfo{addr: addrInfo},
```

Comment:
We persist the new address in our map of addresses associated with this instance